nistats/design_matrix.py

```diff
@@ -450,6 +450,54 @@ def plot_design_matrix(design_matrix, rescale=True, ax=None):
     return ax
 
 
+def create_simple_second_level_design(maps_table, confounds=None,
+                                      main_column_name='contrast'):
+    """Sets up a simple second level design from a maps table.
+
+    Basically its a one column design intended for what would be a simple t
+    test if there are no confounders.
+
+    Parameters
+    ----------
+    maps_table: pandas DataFrame
+        Contains at least columns 'map_name' and 'subject_label'
```

Comment:
I see your point, I made a mistake in the way I remade the function. I am refactoring all this design part after seeing the rest of comments.